common/priority-classes/dkp-priority-classes.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: dkp-critical-priority
+value: 100002000
+globalDefault: false
+description: "This is the highest priority class that is used for critical priority DKP workloads."
+---
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: dkp-high-priority
+value: 100001000
+globalDefault: false
+description: "This is the priority class that is used for high priority DKP workloads."
+---
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: dkp-default-priority
```

Comment:
Would we want to name this default-priority since it'll likely be applied to client specific pods?

Reply:
I'd prefer to label the priority classes that we're creating in such a way that it's obvious they're DKP created classes, but I had a question about whether we want to create this global default class to begin with: https://github.com/mesosphere/kommander-applications/pull/1130#discussion_r1160814770